public/class-fontimator-myaccount.php

```diff
@@ -716,7 +716,7 @@ public function add_address_field_to_edit_account () {
         if ( ! $user_address ) {
             $user_address = new stdClass();
         }
-        if ( ! $user_address->country ) {
+        if (isset($user_address->country) ) {
```

Comment:
Maybe I'm wrong, but this looks like a mistake :)

Reply:
Undefined property: stdClass::$country in plugins/fontimator/public/class-fontimator-myaccount.php on line 719 

you can offer another solution

Reply:
No, I just meant that `! $variable` is the opposite of `isset($variable)`; I just think a `!` needs to be added :)